### PR TITLE
Rebrand Header component

### PR DESCRIPTION
**Pull request by imollov**

### CODE REVIEW

File: src/popup/router/components/Header.vue [hunk 1]
```diff
@@ -88,91 +98,105 @@ export default {
...
+      cursor: pointer;
 
-    button {
-      padding: 0;
-      margin-left: 5px;
+      .count {
+        position: absolute;
+        left: -2px;
+        top: 20%;
+        z-index: 1;
```
Comment (etharner) [inline src/popup/router/components/Header.vue:182]: I don't see any difference after removing it

**Review by etharner — CHANGES_REQUESTED** [review]

File: src/popup/router/components/Header.vue [hunk 2]
```diff
@@ -88,91 +98,105 @@ export default {
 
 <style lang="scss" scoped>
 @import '../../../styles/variables';
+@import '../../../styles/typography';
```
Comment (etharner) [inline src/popup/router/components/Header.vue:101]: In this case we can remove `variables` import as it is already imported in `typography`

**Review by etharner — CHANGES_REQUESTED** [review]

File: src/popup/router/components/Header.vue (hunk 2 above)
Comment (davidyuk) [reply src/popup/router/components/Header.vue:101]: [`@import`](https://sass-lang.com/documentation/at-rules/import) is deprecated, let's use [`@use`](https://sass-lang.com/documentation/at-rules/use) instead. `@use` working differently and probably we won't be needed to remove "duplicate" imports.

Comment (imollov) [reply src/popup/router/components/Header.vue]: Using `@use` instead of `@import` will require making some global changes first.

Comment (mradkov) [reply src/popup/router/components/Header.vue]: Let's log and address this separately and not stall this PR because of this change.

Comment (imollov) [reply src/popup/router/components/Header.vue]: #832

**Review by mradkov — APPROVED** [review]
Changes looking good!

**Pull request merged**